doc/overview/EDITING.md

```diff
@@ -1,125 +1,213 @@
-   ./run_fastsurfer.sh --t1 $datadir/subjectX/t1-weighted-nii.gz \
-                     --sid subjectX --sd $fastsurferdir \
-                     --parallel --threads 4 \
-                     --surf_only
-   ```
-   Note: ```t1-weighted-nii.gz``` would be the original input mri image.
-   
-   Do not forget to give fullpath to the input and output folder unless specified in variables.
-
-## 3. Brainmask Edits: 
-When surfaces go out too far, e.g. they grab dura, you can tighten the mask directly, just edit ```mask.mgz```and start the *surface module*. 
-
-- Step 1: Assuming that you have run the full fastsurfer pipeline once as described in method_1 and successfully produced segmentations and surfaces
-- Step 2: Edit ```mask.mgz``` file in freeview
-- Step 3: Run the pipeline again in order to get the surfaces but before running the pipeline again do not forget to copy all the segmented files in to new input and output directory. 
-   Note: The files in output folder should be pasted in the subjectX folder, the name of subjectX should be the same as it was used in step_1 otherwise it would raise an error of missing files even though the segmentation files exists in output folder.
-   ```bash
-   # Source FreeSurfer
-   export FREESURFER_HOME=/path/to/freesurfer
-   source $FREESURFER_HOME/SetUpFreeSurfer.sh
-
-   # Define data directory
-   datadir=/home/user/my_mri_data
-   fastsurferdir=/home/user/my_fastsurfer_analysis
-
-   # Run FastSurfer
-   ./run_fastsurfer.sh --t1 $datadir/subjectX/t1-weighted-nii.gz \
-                     --sid subjectX --sd $fastsurferdir \
-                     --parallel --threads 4 \
-                     --surf_only
-   ```
+### When to use
+Over- and/or under-segmentation of the white matter: voxels that should be white matter are excluded, or those that should not are included.
+
+### What to do
+Often, these errors should be fixed by [asegdkt_segfile](#asegdkt_segfile) `<subject_dir>/mri/aparc.DKTatlas+aseg.deep.manedit.mgz`, but if that is not successful: 
+1. Open and edit  `<subject_dir>/mri/wm.mgz` and/or `<subject_dir>/mri/filled.mgz`.
+2. [Re-run FastSurfer](#general-process) to propagate the edits.
+
+The manual label 255 indicates a voxel should be included in the white matter and a voxel labeled 1 should not.
+
+See also: <https://surfer.nmr.mgh.harvard.edu/fswiki/FsTutorial/WhiteMatterEdits_freeview>
+
+## Pial surface placement
+
+### When to use
+Over- and/or under-segmentation of the cortical gray matter: voxels that should be gray matter are excluded, or those that should not are included.
+
+This explicitly 
+
+### What to do
+Often, these errors should be fixed in [asegdkt_segfile](#asegdkt_segfile) `<subject_dir>/mri/aparc.DKTatlas+aseg.deep.manedit.mgz`, but if that is not successful:
+1. Open end edit `<subject_dir>/mri/brain.finalsurfs.manedit.mgz` (overwriting values in `<subject_dir>/mri/brain.finalsurfs.mgz`).
+2. [Re-run FastSurfer](#general-process) to fix the pial surface. 
+3. 
```

Comment:
item 3. is missing

Reply:
I think this should just be removed... The docs here does not really have a step, it would just be to check if the results are now better :)
```suggestion
```